### PR TITLE
User sees informational modal when they click the question mark next to the bar chart description

### PR DESCRIPTION
**Pull request by lilyfromseattle**
When user clicks question mark icon
They see a modal

They can click out of the modal by pressing the X, the "Got it" button or clicking on the scrim

Zeplin: zpl.io/2p446zN

Note: We do want to move the modal component into palette, but it's not blocking this ticket.

Copy not finalized yet, can use placeholder

Screen Shot 2019-04-05 at 5 43 19 PM
Screen Shot 2019-04-05 at 5 43 25 PM

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 39383fdbed741658591cf9451e2eb19b9fcf4512

https://deploy-preview-2265--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/PricingContextModal.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            isFixed: false,
+          }}
+        >
+          <Spacer mt={2} />
+          <Serif size="3" color={color("black80")}>
+            This information represents retail prices for works on Artsy, sold
+            and listed. This does not represent auction data.
+          </Serif>
+          <Spacer mt={2} />
+          <Serif size="3" color={color("black80")}>
```
Comment (ds300) [inline src/Apps/Artwork/Components/PricingContextModal.tsx]:
```suggestion
          <Serif size="3" color="black80">
```

File: src/Apps/Artwork/Components/PricingContextModal.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          show={this.state.isModalOpen}
+          title="Pricing distribution"
+          cta={{
+            text: "Got it",
+            onClick: () => this.closeModal(),
+            isFixed: false,
+          }}
+        >
+          <Spacer mt={2} />
+          <Serif size="3" color={color("black80")}>
```
Comment (ds300) [inline src/Apps/Artwork/Components/PricingContextModal.tsx]:
```suggestion
          <Serif size="3" color="black80">
```

File: src/Apps/Artwork/Components/PricingContextModal.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  render() {
+    return (
+      <>
+        <Modal
+          onClose={this.closeModal}
+          show={this.state.isModalOpen}
+          title="Pricing distribution"
+          cta={{
+            text: "Got it",
+            onClick: () => this.closeModal(),
```
Comment (ds300) [inline src/Apps/Artwork/Components/PricingContextModal.tsx]:
```suggestion
            onClick: this.closeModal,
```

File: src/Apps/Artwork/Components/PricingContext.tsx [hunk 4]
```diff
@@ -23,9 +24,12 @@ function PricingContext({ artwork }: PricingContextProps) {
...
         Price
       </Sans>
-      <Sans size="2">
-        Price ranges of {artwork.pricingContext.filterDescription}
-      </Sans>
+      <Flex>
+        <Sans size="2">
+          Price ranges of {artwork.pricingContext.filterDescription}
+        </Sans>
+        <PricingContextModal />
```
Comment (ds300) [inline src/Apps/Artwork/Components/PricingContext.tsx]: Could you add a test for the presence of this in the PricingContext test file? :pray:

**Comment by artsyit** [comment]
:rocket: PR was released in v15.13.6 :rocket:

**Pull request merged**